tools/pkg/process_fakes/pubspec.yaml

```diff
@@ -0,0 +1,24 @@
+# Copyright 2013 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+name: process_fakes
+publish_to: none
+environment:
+  sdk: '>=3.2.0-0 <4.0.0'
+
+# Do not add any dependencies that require more than what is provided in
+# //third_party/pkg, //third_party/dart/pkg, or
+# //third_party/dart/third_party/pkg. In particular, package:test is not usable
+# here.
+
+# If you do add packages here, make sure you can run `pub get --offline`, and
+# check the .packages and .package_config to make sure all the paths are
+# relative to this directory into //third_party/dart
+
+dependencies:
+  process: any
+
+dependency_overrides:
```

Comment:
This will also need to list the transitive dependencies of `package:process`

Reply:
Done, I think?

Reply:
Hmm. `python3 ./tools/pub_get_offline.py` returns exit code 0, I assume that means things are good?

Reply:
Ok hmm, I guess you need a clean CI environment to see the errors :(

Reply:
Filed https://github.com/flutter/flutter/issues/135172

Reply:
Reproduced and fixed!